src/app/containers/Blocks/index.jsx

```diff
@@ -36,11 +26,14 @@ const render = blocks =>
     );
   });
 
-const MainContentContainer = ({ blocks }) => {
-  const renderedContent = render(blocks);
-  return <MainContent>{renderedContent}</MainContent>;
+Blocks.propTypes = {
+  ...mainContentModelPropTypes,
+  blocksToRender: shape({
+    headline: func,
```

Comment:
NB: seems that the `prop-types` lib cannot determine that the value in an object is a React component, so it has to be a `func`, non-ideal but unsure of an alternative.